app/src/main/res/values/strings.xml

```diff
@@ -16,4 +16,7 @@
 
     <string name="hello_first_fragment">Hello first fragment</string>
     <string name="hello_second_fragment">Hello second fragment. Arg: %1$s</string>
```

Comment:
Agora que vi que no meu arquivo acabou ficando alguns itens como esses "Hello first fragment", pode apagar o que não for necessário